examples/standalone/benchmarks/run_on_daint.sh

```diff
@@ -18,43 +23,31 @@ exitError()
     exit $1
 }
 
+# configuration
 SCRIPT=`realpath $0`
-SCRIPTPATH=`dirname $SCRIPT`
-ROOT_DIR="$(dirname "$(dirname "$(dirname "$SCRIPTPATH")")")"
+SCRIPT_DIR=`dirname $SCRIPT`
+ROOT_DIR="$(dirname "$(dirname "$(dirname "$SCRIPT_DIR")")")"
+NTHREADS=12
 
 # check sanity of environment
 test -n "$1" || exitError 1001 ${LINENO} "must pass a number of timesteps"
 timesteps="$1"
-
 test -n "$2" || exitError 1002 ${LINENO} "must pass a number of ranks"
 ranks="$2"
-
+test -n "$3" || exitError 1003 ${LINENO} "must pass a backend"
 backend="$3"
-if [ -z "$3" ] ; then
-    backend="numpy"
-fi
-
-target_dir="$4"
-if [ -z "$4" ] ; then
-    target_dir="$ROOT_DIR"
-fi
-
-data_path="$5"
-if [ -z "$5" ] ; then
-    data_path="/scratch/snx3000/olifu/jenkins/scratch/fv3core_fortran_data/7.2.5/c12_6ranks_standard/"
```

Comment:
I could see this going either way. Having the defaults can be pretty convenient to getting started with a script with a lot of arguments, but can also see how one should probably figure them out pretty soon

Reply:
Agreed. No strong feelings from my side. It felt like this was a left-over from when Tobias was testing things. Note also that this bakes in a fixed serialized data version number, which felt dangerous. Due to these two reasons, I think I would prefer to remove this default.